### PR TITLE
RFC for change to range and replica naming conventions

### PR DESCRIPTION
**Pull request by bdarnell**

### CODE REVIEW

**Comment by tamird** [comment]
I'm in favor

**Comment by tschottdorf** [comment]
also in favor. Renaming variables is cheap and this seems much cleaner.

**Comment by mrtracy** [comment]
This will make a big difference in the clarity of discussion :+1:

**Comment by BramGruneir** [comment]
:+1: or even +:100:

File: docs/RFCS/range_replica_naming.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Summary
+
+Consistently use the word **range** to refer to a portion of the global
+sorted map (i.e. an entire raft consensus group, and **replica** to
+refer to a single copy of the range's data.
+
+# Motivation
+
+We currently use the word "range" informally to refer to both the
+consensus group and the data owned by one member of the group, while
```
Comment (vivekmenezes) [inline docs/RFCS/range_replica_naming.md]: Is there a term we need to use for the the consensus group itself, or are you making it a part of Range itself?

Comment (bdarnell) [inline docs/RFCS/range_replica_naming.md]: The consensus group will be included under the term "range".

**Pull request merged**